### ISSUE TITLE
docker-compose fails

### ISSUE THREAD

**Issue by dl8bh** [google/timesketch#912]
Starting with fresh ubuntu 18.04 server, docker-ce and docker-compose installed leads to the following error:

```

Removing intermediate container e66e89267c2d
 ---> 2ad41408403c
Step 16/23 : RUN cp /tmp/timesketch/timesketch.conf /etc
 ---> Running in a877c62ebe8e
cp: cannot stat '/tmp/timesketch/timesketch.conf': No such file or directory
ERROR: Service 'timesketch' failed to build: The command '/bin/sh -c cp /tmp/timesketch/timesketch.conf /etc' returned a non-zero code: 1
```

**Comment by deeso** [comment]
I am also experiencing a similar issue.

**Comment by deeso** [comment]
Is this line:
``` RUN cp /tmp/timesketch/timesketch.conf /etc ```

supposed to be:

``` RUN cp /tmp/timesketch/data/timesketch.conf /etc ```?

**Comment by deeso** [comment]
This may not be entirely useful, but I had to make several modifications to the `Dockerfile`, `docker-compose.yml`, `tasks.py`, and `timesketch.conf` files.

I captured my work in the following gist.  **Note:** I make no effort to ensure best practices are followed or the service is secure.  Ye be forewarned, arrr!

[Make timesketch work insecurely - gis](https://gist.github.com/deeso/0c2b63676a6b0d12ed2d9b942ab3d6b7)
